cpeval2

```diff
@@ -2063,52 +2082,52 @@ sub parse_output_file {
 
         ## Reseller duplicate username conflict
         # s:Plesk11::RESELLER_CONFLICT:summit
-        if (m{ \A $panel_regex:RESELLER_CONFLICT:(.*) }xms) {
+        if (m{ \A $PANEL_REGEX:RESELLER_CONFLICT:(.*) }xms) {
             push @reseller_username_conflicts, $1;
         }
 
-        if (m{ \A $panel_regex:PHPINI_MEMORY_LIMIT:(\d+) }xms) {
-            $src_phpini_memory_limit = $1;
+        if (m{ \A $PANEL_REGEX:PHPINI_MEMORY_LIMIT:(\d+) }xms) {
+            $src_phpini_memory_limit = length $1 ? $1 : 'NOT SET';
         }
         elsif (m{ \A d:PHPINI_MEMORY_LIMIT:(\d+) }xms) {
-            $dst_phpini_memory_limit = $1;
+            $dst_phpini_memory_limit = length $1 ? $1 : 'NOT SET';
         }
 
-        if (m{ \A $panel_regex:PHPINI_POST_MAX_SIZE:(\d+) }xms) {
-            $src_phpini_post_max_size = $1;
+        if (m{ \A $PANEL_REGEX:PHPINI_POST_MAX_SIZE:(\d+) }xms) {
+            $src_phpini_post_max_size = length $1 ? $1 : 'NOT SET';
         }
         elsif (m{ \A d:PHPINI_POST_MAX_SIZE:(\d+) }xms) {
-            $dst_phpini_post_max_size = $1;
+            $dst_phpini_post_max_size = length $1 ? $1 : 'NOT SET';
         }
 
-        if (m{ \A $panel_regex:PHPINI_UPLOAD_MAX_FILESIZE:(\d+) }xms) {
-            $src_phpini_upload_max_filesize = $1;
+        if (m{ \A $PANEL_REGEX:PHPINI_UPLOAD_MAX_FILESIZE:(\d+) }xms) {
+            $src_phpini_upload_max_filesize = length $1 ? $1 : 'NOT SET';
         }
         elsif (m{ \A d:PHPINI_UPLOAD_MAX_FILESIZE:(\d+) }xms) {
-            $dst_phpini_upload_max_filesize = $1;
+            $dst_phpini_upload_max_filesize = length $1 ? $1 : 'NOT SET';
         }
```

Comment:
Maybe in the future we can do some cleanups for things like this and us a sub to reduce the repetition.

EX:
```
     elsif (m{ \A d:PHPINI_POST_MAX_SIZE:(\d+) }xms) {
           $dst_phpini_post_max_size = length $1 ? $1 : 'NOT SET';
      }
```
Could be:
```
    elsif (m{ \A d:PHPINI_POST_MAX_SIZE:(\d+) }xms) { set( $length ); }
```
```
sub set {
    my $value = shift;
    return length $value ? $value : 'NOT SET';
}
```
Not much of an improvement, really, but it could help?  What do you think?  Overkill?